ui/src/components/common/WalletTransaction.tsx

```diff
@@ -126,21 +126,19 @@ const WalletTransaction = (props: {
     )
 
     async function onSubmit() {
+        if (!token || !custodianId) { return; }
         const key = wrapDamlTuple([operator, party]);
-
         setShowSuccessMessage(false)
 
-        let args = {}
```

Comment:
`daml-react` also got upset when defining `args` up here with no type and using it for two different exercises below.

Reply:
yeah, I have come across this before - it wants all _possible_ values to be the same type 👎  which makes it hard to re-use a variable in this way